fluster/main.py

```diff
@@ -61,9 +62,7 @@ def __init__(self) -> None:
         self.resources_dir = os.path.join(
             os.path.join(os.path.dirname(__file__), ".."), RESOURCES_DIR
         )
-        self.results_dir = os.path.join(
-            os.path.join(os.path.dirname(__file__), ".."), RESULTS_DIR
-        )
+        self.results_dir = os.path.join(gettempdir(), RESULTS_DIR)
```

Comment:
Could you specify more about what you mean by `locally`?

From my point of view results data is temporal.

Reply:
The naming of `results_dir` is a bit miss-leading since it holds the output of the decoding to compute the MD5 when it's calculated correctly. We should rename this variable and update the `--output` argument to explain that it stores the decoder output.
Unless --keep is used, this folder is deleted automatically and given it holds lots of data, it's better to store it in /tmp/ to let the OS due to garbage collection.

Reply:
Done in https://github.com/fluendo/fluster/commit/3b634fc